presto-main/src/main/java/io/prestosql/sql/planner/LogicalPlanner.java

```diff
@@ -347,8 +346,15 @@ private RelationPlan createInsertPlan(Analysis analysis, Insert insertStatement)
                     assignments.put(output, cast);
                 }
             }
+            visibleColumnMetadataBuilder.add(column);
+            insertColumnsBuilder.add(columns.get(column.getName()));
```

Comment:
This builder gets the columns from the `INSERT` clause _and_ all the visible columns not mentioned in the `INSERT` clause which don't have a default value. Please add tests where columns without default value are not in the `INSERT` clause.